File: olo-promotions-sdk-go/pkg/promotions/models/requests/accounts/create_account_request.go
package accounts

// CreateAccountRequest represents a request to create a new user account.
type CreateAccountRequest struct {
	// FirstName is the first name of the user.
	FirstName string `json:"firstName"`

	// LastName is the last name of the user.
	LastName string `json:"lastName"`

	// PhoneNumber is the phone number of the user.
	PhoneNumber string `json:"phoneNumber"`

	// EmailAddress is the email address of the user.
	EmailAddress string `json:"emailAddress"`

	// ExternalIdentifier is a unique external identifier for the user.
	ExternalIdentifier string `json:"externalIdentifier"`
}

File: olo-promotions-sdk-csharp/Olo.Promotions.SDK/Requests/CreateAccountRequest.cs
namespace Olo.Promotions.SDK.Requests
{
    public class CreateAccountRequest
    {
        /// <summary>
        /// First name for the user.
        /// </summary>
        public string FirstName { get; set; }
        /// <summary>
        /// Last name for the user.
        /// </summary>
        public string LastName { get; set; }
        /// <summary>
        /// Phone number for the user.
        /// </summary>
        public string PhoneNumber { get; set; }
        /// <summary>
        /// Email address for the user.
        /// </summary>
        public string EmailAddress { get; set; }
        /// <summary>
        /// Unique external identifier for the user.
        /// </summary>
        public string ExternalIdentifier { get; set; }
    }
}